internal/pkg/levee/levee.go

```diff
@@ -122,3 +203,25 @@ func getArgumentPropagator(c *config.Config, call *ssa.Call) ssa.Node {
 
 	return nil
 }
+
+// callValues collects the ssa.Values that are arguments to an ssa.Call.
+// in particular, it handles variadic arguments to some extent.
+// We don't want to analyze what the variadic argument points to (a slice), but we do
+// want to analyze the values *within* the slice.
+func callValues(c *ssa.Call) (values []ssa.Value) {
+	for _, a := range c.Call.Args {
+		slice, ok := a.(*ssa.Slice)
+		if !ok {
+			values = append(values, a)
+			continue
+		}
+		refs := *slice.X.Referrers()
+		// the last referrer is the slice
+		for i := 0; i < len(refs)-1; i++ {
```

Comment:
I could add an explicit check for it to make sure we avoid panics completely, but it is my understanding that getting an empty slice is impossible. Since the `slice` is an `Arg` of the `Call`, then it will at least be referred to by the `Call`.